corgi/collectors/errata_tool.py

```diff
@@ -152,7 +152,7 @@ def get_erratum_components(self, erratum_id: str):
         }
         """
         variant_to_component_map: typing.DefaultDict[str, list] = defaultdict(list)
-        builds = self.get(f"api/v1/erratum/{erratum_id}/builds_list.json")
+        builds = self.get(self._get_url_by_erratum_type(erratum_id, is_rpm))
```

Comment:
This is minor but it's a little nicer to have code like this split across two lines, so that if we get a traceback it's easier to see which part failed.
```suggestion
        builds_url = self._get_url_by_erratum_type(erratum_id, is_rpm)
        builds = self.get(builds_url)
```

Reply:
ok